actionpack/lib/action_controller/metal/redirecting.rb

```diff
@@ -64,6 +64,12 @@ def redirect_to(options = {}, response_options = {})
       self.response_body = "<html><body>You are being <a href=\"#{ERB::Util.unwrapped_html_escape(response.location)}\">redirected</a>.</body></html>"
     end
 
+    # Soft deprecated alias for <tt>redirect_back_or_to</tt> where the fallback_location location is supplied as a kwarg instead
```

Comment:
If it is soft deprecated should it be included in the documentation site? If so I think we should expand `kwarg` to `keyword argument`. But I think we should `# :nodoc:` it.

Reply:
I think it should be included, because if you're working on a code base where it's used, and you then go to look it up, you should see a) that it exists (and what the signature is), and b) that there's a newer version you can use instead.

Will change to "keyword argument" 👍

Reply:
Random passerby commenting on `kwarg` vs `keyword argument` - I didn't realize that kwarg was a keyword argument until seeing this PR and how it was being used linguistically. Even just adding a hyphen would help immensely if there are still locations where brevity are preferred: `kw-arg` (helps readers intuit that there's two words to be expanded).